static/app/utils/formatters.tsx

```diff
@@ -308,6 +309,8 @@ export function formatAbbreviatedNumber(number: number | string) {
   return number.toLocaleString();
 }
 
-export function formatRate(value: number, rateUnit?: string) {
-  return `${value} /${rateUnit ?? 's'}`;
+export function formatRate(value: number, unit?: RateUnits) {
+  return `${formatAbbreviatedNumber(value)}${
+    RATE_UNIT_LABELS[unit ?? RateUnits.PER_SECOND]
+  }`;
```

Comment:
```suggestion
export function formatRate(value: number, unit: RateUnits = RateUnits.PER_SECOND) {
  return `${formatAbbreviatedNumber(value)}${
    RATE_UNIT_LABELS[unit]
  }`;
```

l: I think this signals intent a lot better than having the nullish coalescing operator

Reply:
Agreed!